A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_UpperECAM.js

```diff
@@ -975,6 +1013,22 @@ var A320_Neo_UpperECAM;
                             return (this.getCachedSimVar("L:AIRLINER_FLIGHT_PHASE", "Enum") == 6) && (this.getCachedSimVar("RADIO HEIGHT", "feet") < 800);
                         }
                     },
+                    {
+                        message: "AP OFF",
+                        style: "fail-3",
+                        important: true,
+                        isActive: () => {
+                            return (this.getAPWarning(3) == true);
+                        }
+                    },
+                    {
+                        message: "A/THR OFF",
+                        style: "fail-2",
+                        important: true,
+                        isActive: () => {
+                            return (this.getATHRWarning(3) == true);
```

Comment:
`== true` is unnecessary here. You can remove it along with the parentheses surrounding the whole thing.

Reply:
i'll change that